### DISCUSSION TITLE
Could this allow a Hand Held to act as a Repeater?

### DISCUSSION THREAD

**Discussion by MikeGriffinReborn** [kamilsss655/ESPRI#68] [Ideas]
Could this somehow be used to all Hand Held Radio's to act as repeaters and convey a message say from Florida to Missouri over a network of simplex HT's with this hardware attached? I've been wondering if it would be possible for folks with hand helds and nice 25 to 50 foot antennas to act as repeaters if they had too?

**Comment by MikeGriffinReborn** [comment]
I just looked into the NuNu stuff, incredible, this is exactly what I've been looking for. I've been working on a site I hope to finish in the next several months and although it will have all of the repeaters on it my real goal is to allow Hand Held Ham Radio operators with decent attenna's to register as a Simplex Node, and somehow build a network that could be used if all of the Repeaters were to go down. This I think is exactly what I've been looking for. I have two Quansheng's, currently I have the IJV firmware on it but I just might have to switch now and do some experimentation with NUNU.

My site is https://whiskeyniner.com 

I'm terribly excited about what you guys are doing here !!

**Reply by kamilsss655** [reply]
Hi. Based on your description you might be interested in custom K5 firmware which allows mesh messaging.
More info: https://github.com/kamilsss655/uv-k5-firmware-custom/wiki/43-%E2%80%90-Mesh-network

**Reply by MikeGriffinReborn** [reply]
Yes, I was just looking at that, I'm going to do a video on that on my YouTube channel this weekend. Might add that to my website. My main goal for creating this website was to create a way to register simplex connections with Antenna's that could be used to transmit messages across the country, this is very interesting indeed.

**Reply by MikeGriffinReborn** [reply]
So, I just watched a very good video by M7FRS on you tube demonstrating how NUNU works, super cool. One thing I was wondering (probably a pipe dream) could we input our Callsign into the Radio as a setting and then send the message to a particular callsign? I'm not 100% sure how this works but let's assume there are tons of Quanshengs with the proper firmware installed. I send a message from Florida to to a callsign of someone I know in Missouri. It becomes almost like the www in that it forwards the message to every node it can reach only once, eventually, hopefully, the message reaches the person in Missouri.

Anyway, sorry for pestering you, I have much to read on this
